en/simulation/ignition_gazebo.md

```diff
@@ -1,11 +1,11 @@
-# Ignition Gazebo Simulation
+# Gz Simulation
+:::note
+Gz, a.k.a Gazebo Classic, previously known as 'Gz', is the only supported version of Gazebo for Ubuntu 22.04 and onwards. Therefore it is strongly recommended to use the Gz, instead of the classic Gazebo for simulations.
```

Comment:
> Gz, a.k.a Gazebo Classic, previously known as 'Gz'.

1. Which is it? Gz or Gazebo classic? This implies both, or either.
2. Do you mean "previously Ignition Gazebo"?
3. If the name is Gz, will that affect naming of repositories. IMO these should all match same naming convention. I'd rather call it Gazebo Classic because Gz is an overloaded term - i.e. we have gz server somewhere.
4. Has that change happened already in codelines - if so, what is the PR?

Reply:
Gz is not Gazebo Classic

Gazebo Classic is the old one

Gazebo is the new one (Gazebo Garden, previously called Ignition)